match/lib/match/spaceship_ensure.rb

```diff
@@ -53,14 +53,17 @@ def certificate_exists(username: nil, certificate_id: nil)
       UI.user_error!("To reset the certificates of your Apple account, you can use the `fastlane match nuke` feature, more information on https://docs.fastlane.tools/actions/match/")
     end
 
-    def profile_exists(username: nil, uuid: nil)
+    def profile_exists(username: nil, uuid: nil, return_if_missing: false)
       found = Spaceship.provisioning_profile.all.find do |profile|
         profile.uuid == uuid
       end
 
       unless found
         UI.error("Provisioning profile '#{uuid}' is not available on the Developer Portal")
         UI.error("for the user #{username}")
+
+        return false if return_if_missing
```

Comment:
this would be a chance to tell the user about the parameter they can set, right?